### ISSUE TITLE
nvim library

### ISSUE THREAD

**Issue by Kethku** [Olical/aniseed#9]
It looks like the majority of the usefulness of the nvim.lua library is now present in base vim https://neovim.io/doc/user/lua.html#vim.g would it make sense to remove this unneeded extra dependency now?

**Comment by Olical** [comment]
Hmm I did look at this before but I wasn't sure which parts were _not_ added, so I kinda enjoyed using it since I knew I was getting the full experience. Since my time is VERY limited and spread pretty thin right now I just decided that it wasn't worth the investigation and that using the fully featured one where I knew I had access to everything was the best bet.

like that doesn't add the ex helpers or mappings through to `nvim_call_function` via `nvim.call_function`, right? And I use those _all_ the time.
